robottelo/cli/factory.py

```diff
@@ -1560,7 +1560,7 @@ def make_template(options=None):
     # Assigning default values for attribute
     args = {
         'audit-comment': None,
-        'file': f'/tmp/{gen_alphanumeric()}',
+        'file': f'/root/{gen_alphanumeric()}',
```

Comment:
Would /var/tmp solve the issue as well?

Reply:
Reading through the linked issue there is some back and forth about this, I think we're safer avoiding both.

https://community.theforeman.org/t/temporary-directories-cleanup/22698/6

Reply:
Mike is right, lzap explicitly states in the community forum that they have even separated the `/var/tmp` directory via namespaces. As Mike mentioned, it is safer to avoid both `/tmp` and `/var/tmp`.